CMakeLists.txt

```diff
@@ -235,6 +235,17 @@ if (NANOGUI_BUILD_SHARED)
   target_link_libraries(nanogui ${NANOGUI_EXTRA_LIBS})
 endif()
 
+install(
+  TARGETS nanogui
+  LIBRARY DESTINATION lib
+  ARCHIVE DESTINATION lib
+  )
+
+install(
+  DIRECTORY include/nanogui DESTINATION include
```

Comment:
Do these files get installed to <destination>/include or <destination>/include/nanogui? The latter is what would be intended by the library.